switchbot_ros/package.xml

```diff
@@ -13,14 +13,15 @@
   <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
   <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
 
+  <build_depend>actionlib_msgs</build_depend>
   <build_depend>message_generation</build_depend>
+  <build_depend>std_msgs</build_depend>
 
   <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-requests</exec_depend>
   <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-requests</exec_depend>
   <exec_depend>actionlib</exec_depend>
   <exec_depend>message_runtime</exec_depend>
   <exec_depend>rospy</exec_depend>
-  <exec_depend>std_msgs</exec_depend>
```

Comment:
I am not confident, but `build_depends` and `exec_depends` are independent, so adding `std_msgs` in `build_depends` does not mean the generated binary file depends on `std_msgs`

Reply:
so its better to add it in both `build_depend` and `run_depend`?

Reply:
Anyway, We need to check if we can generate deb file and install correctly.

Reply:
I added some depends